system/ee/ExpressionEngine/Addons/channel/mod.channel.php

```diff
@@ -4025,6 +4029,8 @@ public function category_heading()
             }
 
             if ($valid == 'y') {
+                $qstring = preg_replace("/(.*?)\/" . preg_quote($this->reserved_cat_segment) . "\//i", '', '/' . $qstring);
```

Comment:
Is there some particular reason why this line of code has been moved down here?

Reply:
Just because the `$qstring` isn't use anywhere until this block. I can move it back.

Reply:
It might be just safer to keep it where it was - so can you revert?

Reply:
Sure!